packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -200,12 +237,17 @@ class ChatMessage extends UIComponent<ReactProps<ChatMessageProps>, ChatMessageS
               },
             })}
 
+            {reactionsPosition === 'start' && reactions && this.renderReactions(styles)}
+
             {Box.create(content, {
               defaultProps: {
                 className: ChatMessage.slotClassNames.content,
                 styles: styles.content,
               },
             })}
+
+            {reactionsPosition === 'end' && reactions && this.renderReactions(styles)}
```

Comment:
```jsx
const reactionsElement = reactions && this.renderReactions(styles)
```

```suggestion
            {reactionsPosition === 'end' && reactionsElement}
```

Reply:
I've added the reactions condition here, because if I don't have it, I will create an empty Box there with some styles... I can move the condition to the function though... Now that I think about it, maybe ReactionGroup would be a good way to go...

Reply:
I've added the `ReactionGroup` component, and this has been resolved with it.